pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/manager/config.go

```diff
@@ -58,6 +65,12 @@ metadata:
   namespace: system
   labels:
     control-plane: controller-manager
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
+    app.kubernetes.io/managed-by: k8s.io
```

Comment:
```suggestion
    app.kubernetes.io/managed-by: kustomize
```

Reply:
resolved in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/de5ad3beb0988a49f533ee8c1a8c87fd82e7bf02